tests/block_test_util.go

```diff
@@ -330,6 +330,12 @@ func (t *BlockTest) validatePostState(statedb *state.StateDB) error {
 		if nonce2 != acct.Nonce {
 			return fmt.Errorf("account nonce mismatch for addr: %s want: %d have: %d", addr, acct.Nonce, nonce2)
 		}
+		for k, v := range acct.Storage {
+			v2 := statedb.GetState(addr, k)
+			if v2 != v {
+				return fmt.Errorf("account storage mismatch for addr: %s, slot: %x, want: %x, have: %x", addr, k, v, v2)
```

Comment:
```suggestion
			
			if have := statedb.GetState(addr, k); have != v {
				return fmt.Errorf("account storage mismatch for addr: %s, slot: %x, want: %x, have: %x", addr, k, v, have)
```

Reply:
I went with `v2` to match the style of the surrounding code, but I also prefer `have`. What do you think about renaming the other variables, e.g., `code2` -> `have_code`, for consistency?